learners-guild/README.md

```diff
@@ -97,22 +121,24 @@ The easiest way to test changes locally (that I've found so far) is to:
 
 4. At this point, any changes you make in your cloned repository will cause the Meteor server to restart and the client to reload (which, on Meteor 1.2, takes forever). Once you're confident that things are working, you should bump the version number (using good [semver][semver] practices), publish the package to meteor, and reverse the process you did above:
 
-        $ meteor remove learnersguild:rocketchat-lg-slash-commands
-        $ rm packages/rocketchat-lg-slash-commands
-        $ cd ../rocketchat-lg-slash-commands
-        #
-        # BUMP VERSION IN package.js
-        #
-        $ meteor publish
-        $ cd ../Rocket.Chat
-        $ meteor add learnersguild:rocketchat-lg-slash-commands
+```bash
+meteor remove learnersguild:rocketchat-lg-slash-commands
```

Comment:
I'm just changing this from indentation to backticks. Would you like m to remove the while section?